docs/modules/ROOT/pages/index.adoc

```diff
@@ -13,7 +13,7 @@
 *** xref:mgmt_services/vpc_manager.adoc[VPC Manager]
```

Comment:
We will need a new route_manager_l3_neighbors_update_5.PNG to remove the grammar check warning.

Reply:
The PNG is recaptured without grammar check warning.